src/bindings/python/tests/mock/pyngraph_fe_mock_api/CMakeLists.txt

```diff
@@ -5,31 +5,18 @@
 set(PYBIND_FE_NAME "pybind_mock_frontend")
 
 set(PYBIND_FE_SRC ${CMAKE_CURRENT_SOURCE_DIR}/pyngraph_mock_frontend_api.cpp)
+set(pyversion python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
 
 source_group("src" FILES ${PYBIND_FE_SRC})
 
 pybind11_add_module(${PYBIND_FE_NAME} MODULE NO_EXTRAS ${PYBIND_FE_SRC})
 
-set(DEPENDENCIES openvino_mock_py_frontend openvino::runtime)
-set(DEFINITIONS IMPLEMENT_OPENVINO_API)
-
-if (ENABLE_OV_ONNX_FRONTEND)
-    list(APPEND DEPENDENCIES openvino::frontend::onnx)
-    list(APPEND DEFINITIONS ENABLE_OV_ONNX_FRONTEND)
-endif()
-
-if (ENABLE_OV_TF_FRONTEND)
-    list(APPEND DEPENDENCIES openvino::frontend::tensorflow)
-    list(APPEND DEFINITIONS ENABLE_OV_TF_FRONTEND)
-endif()
-
-if (ENABLE_OV_PADDLE_FRONTEND)
-    list(APPEND DEPENDENCIES openvino::frontend::paddle)
-    list(APPEND DEFINITIONS ENABLE_OV_PADDLE_FRONTEND)
-endif()
-
-target_compile_definitions(${PYBIND_FE_NAME} PRIVATE ${DEFINITIONS})
-target_link_libraries(${PYBIND_FE_NAME} PRIVATE ${DEPENDENCIES})
+target_compile_definitions(${PYBIND_FE_NAME} PRIVATE
+        IMPLEMENT_OPENVINO_API
```

Comment:
BTW, I wonder whether `IMPLEMENT_OPENVINO_API` is needed? It's used only for code which exports something outside, while FE imports OpenVINO library symbols

Reply:
It looks like is not required any more, will remove.